src/com/ableton/Pyenv.groovy

```diff
@@ -122,15 +122,15 @@ class Pyenv implements Serializable {
     List commands = ["export PYENV_ROOT=${pyenvRoot}"]
     if (script.env.OS != 'Windows_NT') {
       commands += [
-        "export PATH=\$PYENV_ROOT/bin:\$PATH",
+        "export PATH=\$PYENV_ROOT/bin:/usr/bin:/bin",
         'eval "\$(pyenv init --path)"',
         'eval "\$(pyenv init -)"',
       ]
     } else {
       String posixPyenvRoot = pyenvRoot[1] == ':' ?
         "/${pyenvRoot[0].toLowerCase()}/${pyenvRoot.substring(3)}" : pyenvRoot
       commands.add(
-        "export PATH=${posixPyenvRoot}/shims:${posixPyenvRoot}/bin:\$PATH"
+        "export PATH=${posixPyenvRoot}/shims:${posixPyenvRoot}/bin:/usr/bin:/bin"
```

Comment:
What is in `/usr/bin` and `/bin` on Windows?

Reply:
The contents of `C:\Program Files\git\usr\bin` and `C:\Program Files\git\bin`, respectively. Those directories seem to have the same contents, FWIW.